projects/lux/src/lib/helperFns.ts

```diff
@@ -22,6 +22,21 @@ export const isValidUrl = (value: string): boolean => {
   return re.test(String(value).toLowerCase().trim());
 };
 
+export const isValidColor = (value: string): boolean => {
+  value = String(value).toLowerCase();
+  // valid values for CSS color property, yet not valid colors by themselves
+  if (
+    value === 'currentcolor' ||
+    value === 'inherit' ||
+    value === 'initial' ||
+    value === 'revert' ||
+    value === 'unset'
+  ) {
+    return false;
+  }
+  return CSS.supports('color', value);

```

Comment:
This is smart!
https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports
I didn't know it.